client/src/modules/login/login.html

```diff
@@ -3,15 +3,26 @@
     <div class="row">
       <div class="col-md-4 col-md-offset-4 col-xs-12">
 
-        <div ng-if="!LoginCtrl.installationExist" class="text-center" style="margin-top : calc(50vh - 190px); max-width: 400px; margin-right: auto; margin-left: auto;">
+        <div 
+          ng-if="!LoginCtrl.installationExist && LoginCtrl.finishInstallationChecking && !LoginCtrl.installationCheckingError" 
```

Comment:
Do you need the `LoginCtrl.installationCheckingError`?  It looks like this code would work fine without it.

Reply:
When I try to stop redis and launch bhima, I have the installation button displayed.
The checking operation in this case can't succeed because redis is down.
@jniles , try to create an error (network,...) and see what will happen

Reply:
Aha.  Interesting.  Okay 👍